Wikipedia/Code/SectionEditorViewController.swift

```diff
@@ -668,12 +668,18 @@ extension SectionEditorViewController: SectionEditorInputViewsControllerDelegate
                 }
                 editLinkViewController.delegate = self
                 let navigationController = WMFThemeableNavigationController(rootViewController: editLinkViewController, theme: self.theme)
+                if #available(iOS 13.0, *) {
+                    navigationController.modalPresentationStyle = .automatic
+                }
                 navigationController.isNavigationBarHidden = true
                 self.present(navigationController, animated: true)
             } else {
                 let insertLinkViewController = InsertLinkViewController(link: link, siteURL: siteURL, dataStore: dataStore)
                 insertLinkViewController.delegate = self
                 let navigationController = WMFThemeableNavigationController(rootViewController: insertLinkViewController, theme: self.theme)
+                if #available(iOS 13.0, *) {
+                    navigationController.modalPresentationStyle = .automatic
```

Comment:
Is there a chance of either of these being affected by https://phabricator.wikimedia.org/T232604 ?

Reply:
Yes, articles syncing or losing connection could cause the alerts to show up